functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/DialogAction.java

```diff
@@ -247,6 +252,96 @@ public void checkAutomaticSync() {
 
 	}
 
+	public class MavenDialogAction {
+
+		public CheckBox getAddPluginTypeSuffixForMavenProjectContextRoot() {
+			return _mavenDialog.getAddPluginTypeSuffixForMavenProjectContextRoot();
+		}
+
+		public Button getApplyBtn() {
+			return _mavenDialog.applyBtn();
+		}
+
+		public TextInGroup getDefaultArchetypes() {
+			return _mavenDialog.getDefaultArchetypes();
+		}
+
+		public CheckBox getDisableCustomJspValidationChecking() {
+			return _mavenDialog.getDisableCustomJspValidationChecking();
+		}
+
+		public Text getExt() {
+			return _mavenDialog.getExt();
+		}
+
+		public Text getHook() {
+			return _mavenDialog.getHook();
+		}
+
+		public Text getLayoutTemplate() {
+			return _mavenDialog.getLayoutTemplate();
+		}
+
+		public CLable getMaven() {
+			return _mavenDialog.mavenLable();
+		}
+
+		public TextInGroup getMavenProjectConfiguratorOption() {
```

Comment:
Should be Options, check eclipse and please make sure every one is correct, thanks